File: domain/src/main/java/com/ractoc/eve/domain/fleetmanager/RegistrationModel.java
package com.ractoc.eve.domain.fleetmanager;

import io.swagger.annotations.ApiModel;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
@ApiModel(value = "Registration Model", description = "Contains the Fleet Registration model information")
public class RegistrationModel {

    private Integer fleetId;
    private Integer characterId;
    private String name;
    private Integer roleId;

    private String start;
    private String end;

    private String fleetOwner;
    private String corporationName;
    private FleetModel fleet;

}

File: angular/MineBuildSell/src/app/shared/model/Registration-model.ts
import {FleetModel} from './fleet-model';

export class RegistrationModel {

  constructor(public fleetId: number,
              public characterId: number,
              public fleet: FleetModel,
              public name: string,
              public characterName: string,
              public loadingIcon: boolean,
              public icon: string,
              public roleId: number,
              public start: string,
              public end: string) {
  }

}
